File: 240517/ab 사이에 없는 c/c-between-a-and-b-2.js
const fs = require('fs');
const [a, b, c] = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let hasMultiplesOfC = false;
for(let i = a; i <= b; i++){
    if(i % c === 0){
        hasMultiplesOfC = true;
        break;
    }
}

console.log(hasMultiplesOfC ? "NO" : "YES");

File: 240517/ab 사이에 없는 c/c-between-a-and-b-2.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();

        boolean hasMultiplesOfC = false;
        for(int i = a; i <= b; i++){
            if(i % c == 0){
                hasMultiplesOfC = true;
                break;
            }
        }

        System.out.println(hasMultiplesOfC ? "NO" : "YES");
    }
}